gm4_ambrosia/data/ambrosia/functions/hive/find_flowers.mcfunction

```diff
@@ -0,0 +1,31 @@
+# @s = beehive that is being processed
+# at @s
+# run from check_hive_health
+scoreboard players set #inverse gm4_bee_data 100
+#find the numbers of flowers and air blocks around the hive
+execute store result score flower_count gm4_bee_data run clone ~3 ~1 ~3 ~-3 ~-1 ~-3 ~-3 ~-1 ~-3 filtered #ambrosia:flowers force
+execute store result score air_count gm4_bee_data run clone ~3 ~1 ~3 ~-3 ~-1 ~-3 ~-3 ~-1 ~-3 filtered #ambrosia:air force
+
+# get flower to air ratio (100 being 1:1)
+scoreboard players operation flower_ratio gm4_bee_data = flower_count gm4_bee_data
+scoreboard players operation flower_ratio gm4_bee_data *= #10000 gm4_bee_data
```

Comment:
Multiplying by `#10000` and then dividing by `#100` is just multiplying by `#100`